packages/grafana-data/src/types/panel.ts

```diff
@@ -36,6 +36,7 @@ export interface PanelProps<T = any> {
   timeZone: TimeZone;
   options: T;
   onOptionsChange: (options: T) => void;
+  fieldConfig?: FieldConfigSource;
```

Comment:
Stores panel's field config, setter added below. Used i.e. in stat and piechart panel. Need to make sure the defaults are respected.